Bottom: 91f74e51339a36da5305ac9d82bc04061a1495e0
Top:    6a68b77fab118a10cfd25f7935e2dcc5968c4d87
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-12-08 10:36:33 +0800

dev: reproducer

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/net/core/dev.c b/net/core/dev.c
index 6bea893..08c5a85 100644
--- a/net/core/dev.c
+++ b/net/core/dev.c
@@ -7674,6 +7674,11 @@ int register_netdevice(struct net_device *dev)
 	else if (__dev_get_by_index(net, dev->ifindex))
 		goto err_uninit;
 
+	if (!strcmp(dev->name, "tap0")) {
+		printk("trigger!\n");
+		goto err_uninit;
+	}
+
 	/* Transfer changeable features to wanted_features and enable
 	 * software offloads (GSO and GRO).
 	 */
